README.md

```diff
@@ -5,8 +5,8 @@
 ## Description
 Implement Products catalog following [this design](https://www.figma.com/file/uEetgWenSRxk9jgiym6Yzp/Phone-catalog-redesign?node-id=1%3A2).
 
-Use [products](https://mate-academy.github.io/react_phone-catalog/api/products.json)
-and [product details](https://mate-academy.github.io/react_phone-catalog/api/products/motorola-xoom.json)
+Use [products](https://mate-academy.github.io/react_phone-catalog/api/phones.json)
+and [product details](https://mate-academy.github.io/react_phone-catalog/api/phones/apple-iphone-7-32gb-black.json)
```

Comment:
Why do these links use `/phones` instead of `/products`?
Logically it is wrong, because we can display not only phones